app-modules/ai/src/Http/Controllers/QnaAdvisors/ShowAdvisorController.php

```diff
@@ -71,6 +72,12 @@ public function __invoke(Request $request, QnaAdvisor $advisor): JsonResponse
                 'name' => $advisor->name,
                 'description' => $advisor->description,
                 'avatar_url' => $advisor->getFirstTemporaryUrl(now()->addHour(), 'avatar') ?: null,
+                'title_text_color' => QnaAdvisorCardViewFeature::active() && $advisor->title_text_color ? $advisor->title_text_color : '#000000',
```

Comment:
It would be more terse to use null-coalescing syntax here and for all the others below it.

```suggestion
                'title_text_color' => QnaAdvisorCardViewFeature::active() && $advisor->title_text_color ?? '#000000',
```

Reply:
Oh I see! You are right. Sounds good